src/components/auth/torus/sdk/strategies.js

```diff
@@ -31,7 +31,15 @@ export const LoginStrategy = {
 class AbstractLoginStrategy {
   constructor(torus, config) {
     this.torus = torus
-    this.config = config
+
+    if (isMobileNative) {
```

Comment:
this should be moved to the torus.native.js to the triggerLogin method

Reply:
@MarianoAmado

Reply:
@serdiukov-o-nordwhale 
I don't see how moving this to torus.native.js could improve the solution, as there we just call the module's triggerLogin with the received `loginOptions` object, and changing this value would be harder as the object could have different structures and no obvious way of verifying if it contains `auth0Domain`.

Also i think it makes more sense to have the strategies having the correct values for native than fixing or adapting them on our torus sdk wrapper; an alternative to this could be to do this in config.js (check isMobileNative & remove the  'https://' part ) as this is the only place we are using config.auth0Domain